tests/testthat/test-3-prioritise_and_diagnose.R

```diff
@@ -0,0 +1,55 @@
+test_that("with known input, runs without error", {
+  config <-
+    list(
+      directories = list(
+        dir_input = test_path("test-data", "input"),
+        dir_prepared_abcd = test_path("test-data", "output", "prepared_abcd"),
+        dir_matched_loanbooks = test_path("test-data", "output", "matched_loanbooks"),
+        dir_prioritized_loanbooks_and_diagnostics = test_path("test-data", "output", "prioritized_loanbooks_and_diagnostics")
+      ),
+      project_parameters = list(
+        scenario_source = "weo_2022",
+        scenario_select = "nze_2050",
+        region_select = "global",
+        start_year = 2022L,
+        time_frame = 5L,
+        by_group = "group_id"
+      ),
+      sector_split = list(
+        apply_sector_split = FALSE,
+        sector_split_type = "equal_weights"
+      ),
+      matching = list(
+        params_match_name = list(
```

Comment:
I don't think this section should be needed here

Reply:
`run_match_prioritize()` will error if `sector_split_type` is not set

https://github.com/RMI-PACTA/pacta.multi.loanbook/blob/813fdc4537972e0fbe53762c03c4be3f763fdeed/R/run_match_prioritize.R#L58

probably should fix that... will open new issue

Reply:
#189

Reply:
I actually meant the `matchting` section, not `sector_split_type`

Reply:
Oops. Make a new issue and I'll look at it. I found a bunch of places where some input was required even if a part of the process was being skipped